docs/install_guides/installing-openvino-pip.md

```diff
@@ -73,23 +73,28 @@ python -c "from openvino.runtime import Core"
 
 If installation was successful, you will not see any error messages (no console output).
 
-## Installing OpenVINO Development Tools
-
-OpenVINO Development Tools include Model Optimizer, Benchmark Tool, Accuracy Checker, Post-Training Optimization Tool and Open Model Zoo tools including Model Downloader. If you want to install OpenVINO Development Tools, OpenVINO Runtime will also be installed as a dependency, so you don't need to install OpenVINO Runtime separately.
+Congratulations! You finished installing OpenVINO Runtime. Now you can start exploring OpenVINO's functionality through Jupyter Notebooks and sample applications. See the [What's Next?](#whats-next) section to learn more!
```

Comment:
Again, the `[What's Next?](#whats-next)` will not work. We need to change it to the `<a>` tags.

Reply:
Okay, I changed the link. I changed it to use the direct hotlink, like Karol showed:

```
See the [What's Next](https://docs.openvino.ai/2022.2/openvino_docs_install_guides_installing_openvino_pip.html#what-s-next) section to learn more!
```

Let me know if I should change it to use <a> tags instead!

Reply:
No that's not what I meant. It should be similar to this comment: https://github.com/openvinotoolkit/openvino/pull/13156#issuecomment-1258936466 
I've corrected it. :)

Reply:
Thanks Yuan!